VERSION

```diff
@@ -1 +1 @@
-0.46.0
+0.46.0
```

Comment:
```suggestion
0.46.0

```

ugh, we lost the newline in my attempt to revert something from local testing. This fixes that, so this file should completely drop out of the diff.

Since we're planning to archive this repo soon (#1160 ) I won't add a PR with the `pre-commit` hook to fix trailing newlines.